crates/turbopath/src/absolute_system_path_buf.rs

```diff
@@ -245,6 +261,11 @@ impl AbsoluteSystemPathBuf {
     pub fn open(&self) -> Result<fs::File, PathError> {
         Ok(fs::File::open(&self.0)?)
     }
+
+    pub fn to_realpath(&self) -> Result<Self, PathError> {
```

Comment:
What does realpath mean in this context?

Reply:
`man realpath` -> "realpath – returns the canonicalized absolute pathname"

It's a standard unix tool that resolves all symlinks and gives an absolute path.